horace_core/sqw/PixelData/@PixelDataFileBacked/private/binary_op_pixels_.m

```diff
@@ -48,25 +48,27 @@
     % TODO: #975 loop have to be moved level up calculating image in single
     % loop too
     num_pages= obj.num_pages;
+    this_sigvar = sigvar();
+    other_sigvar = sigvar();
     for i = 1:num_pages
         obj.page_num = i;
         data = obj.data;
 
-        [start_idx, end_idx] = obj.get_page_idx_(i);
+        this_sigvar.sig_var = obj.sig_var;
+        % this is different from above
```

Comment:
I don't understand this; what above are you referring to? Please give the line number. And, how is it different?

Reply:
hopefully clarified